src/__tests__/renderer.test.tsx

```diff
@@ -13,9 +13,34 @@ import {
   Mention,
   ProgressBar,
   Message,
+  MessageText,
 } from '..';
 
 describe('slack jsx', () => {
+  it('message with complex fallback text', () => {
+    const message = (
+      <Message
+        text={
```

Comment:
would it be more obvious to call this `fallbackText` or `notificationText` or `altText`?

Reply:
any of those would probably be better, i'm going to go with `altText` for now